cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineScheduleButton.js

```diff
@@ -43,7 +41,6 @@ export default class PipelineScheduleButton extends Component {
 
   state = {
     showScheduler: false,
-    showConfigModeless: false,
```

Comment:
I believe there are still other references to `this.state.showConfigModeless` in this component